### PR TITLE
add doc for broadcast_to 

### PR DESCRIPTION
**Pull request by sandyhouse**
1. add doc for broadcast_to;
2. remove paddle.disable_static in expand api doc

### CODE REVIEW

File: doc/paddle/api/paddle/tensor/manipulation/broadcast_to_cn.rst [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    - name (str，可选) - 操作的名称(可选，默认值为None）。更多信息请参见 :ref:`api_guide_Name` 。
+
+返回
+:::::::::
+``Tensor`` ，数据类型与 ``x`` 相同。
+
+代码示例
+:::::::::
+
+.. code-block:: python
```
Comment (TCChenlong) [inline doc/paddle/api/paddle/tensor/manipulation/broadcast_to_cn.rst:25]: 示例代码需要和英文一致~

File: doc/paddle/api/paddle/tensor/manipulation/expand_cn.rst [hunk 2]
```diff
@@ -27,7 +27,6 @@ expand
        import paddle
        import numpy as np
                
-       paddle.disable_static()
```
Comment (TCChenlong) [inline doc/paddle/api/paddle/tensor/manipulation/expand_cn.rst:30]: 同上

File: doc/paddle/api/paddle/tensor/manipulation/broadcast_to_cn.rst (hunk 1 above)
Comment (sandyhouse) [reply doc/paddle/api/paddle/tensor/manipulation/broadcast_to_cn.rst:25]: done.

File: doc/paddle/api/paddle/tensor/manipulation/expand_cn.rst (hunk 2 above)
Comment (sandyhouse) [reply doc/paddle/api/paddle/tensor/manipulation/expand_cn.rst:30]: done.

**Review by TCChenlong — APPROVED** [review]
LGTM

**Pull request merged**